lona/routing.py

```diff
@@ -179,15 +179,11 @@ def _reverse(self, name, *args, **kwargs):
         if route.path:
             return route.path
 
-        key_error = None
-
         try:
             return route.format_string.format(*args, **kwargs)
 
         except KeyError as e:
-            key_error = e
-
-        raise ValueError('missing URL arg: {} '.format(key_error.args[0]))
+            raise ValueError('missing URL arg: {} '.format(e.args[0]))
```

Comment:
This is actually on purpose to avoid stacking tracebacks

Reply:
Reverted and added comment